tests/integration/s2n_test_constants.py

```diff
@@ -112,6 +112,7 @@
     S2N_TLS11 : "TLSv1.1",
     S2N_TLS12 : "TLSv1.2",
     S2N_TLS13 : "TLSv1.3",
+    None      : "Default",
```

Comment:
would it better to call this Default or S2N_TLS_DEFAULT instead of None?

Reply:
This specific dict is a mapping of version numbers (ie, S2N_TLS13 is 34) to printed names. The actual version numbers are defined at the beginning of this file. "None" is Python's equivalent of NULL, so this line says to print a missing version number as "Default".

I initially added a S2N_TLS_DEFAULT version number to use, setting its value to "99", but that seemed kind of wrong. I can go back to it, but I think using None captures the meaning better, especially since that's how we handle all the other possible variations on test parameters.

Reply:
`None` can be overloaded but given the need to define a default version somewhere else I think None would be more acceptable here.